packages/spotlight/src/spotlight.js

```diff
@@ -356,7 +356,7 @@ const Spotlight = (function () {
 	}
 
 	function handleWebOSMouseEvent (ev) {
-		if (ev && ev.detail && ev.detail.type === 'Leave') {
+		if (!isPaused() && ev && ev.detail && ev.detail.type === 'Leave') {
```

Comment:
Can we assume that any input that will bring up the vkb will imply Moonstone?  Or, should we perhaps check to see that the type of focused element isn’t an input, instead?

Reply:
I'd rather avoid checking component types, as focus changes simply shouldn't happen when spotlight is paused - which should satisfy most (every?) use-cases. There are, however, other things to consider in a followup ticket to make sure we're covered during focus, blur, pointer enter and leave events, as mentioned above. We can revisit edge-cases then.